x-pack/plugin/esql/qa/server/multi-clusters/src/javaRestTest/java/org/elasticsearch/xpack/esql/ccq/Clusters.java

```diff
@@ -43,6 +44,7 @@ public static ElasticsearchCluster localCluster(ElasticsearchCluster remoteClust
             .setting("cluster.remote.connections_per_cluster", "1")
             .shared(true)
             .setting("cluster.routing.rebalance.enable", "none")
+            .plugin("inference-service-test")
```

Comment:
this plugin is used for testing `semantic_text` - it's an inference service that can create sparse or dense embeddings - that have no actual "semantic" meaning since they are not using a model, but they are supposed to be deterministic.